packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -198,6 +209,18 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
     }
   };
 
+  const expandedLabel = () => {
+    if (isExpanded) {
+      return 'Close';
```

Comment:
These need to be something the consumer can override - consider internationalization where people need to be able to translate their sites into different languages.

Reply:
+1 about `aria-expanded` with a separate `ariaLabel` type prop.

Small nit is that I would prefer this to not be a function (an inline if block should work the same), but either way.